projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-saved-cart.ts

```diff
@@ -301,6 +301,87 @@ export function verifyCartDetails(cart: any) {
     });
 }
 
+export function getCartItem(name: string) {
+  return cy.get('cx-cart-item-list').contains('.cx-item-list-row', name);
+}
+
+export function restoreSavedCart(cart: any) {
+  const restoreSavedCartAlias = interceptRestoreSavedCartEndpoint(cart.code);
+  const getAllSavedCartAlias = interceptGetAllSavedCartEndpoint();
+
+  cy.get('cx-saved-cart-details-action .btn-primary').click();
```

Comment:
```suggestion
  function clickOnPrimaryButton() {
      cy.get('cx-saved-cart-details-action .btn-primary').click();
  }
```
I dont know if it is a Primary, but please change by the proper identifier